build.js

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/node
+const af = require('async-file');
```

Comment:
This file is only used by the dev to generate the file in `dist/`.

But for clarifications, the generated files in `dist/` are to be copied as-is to be used in the PubNub BLOCK Event Handler's. This seems to be a very limited JS environment where I can use some modules to do http, crypto et al.